Dockerfile

```diff
@@ -7,7 +7,7 @@
 # ************************************************************
 # To build, execute `docker-tools <docker/docker_tools.py>` ``build``.
 
-FROM python:3.9-buster
```

Comment:
Nice that we can upgrade a whole operating system with just a couple of characters.

Reply:
Amazing! Even better, the downtime is so minimal -- I build another container, stop the current container, then re-start the new container.